webapp/src/css/sidebar-filter.less

```diff
@@ -99,6 +99,68 @@
       overflow-y: auto;
       overflow-x: hidden;
```

Comment:
You can decrease the margin a bit to give more space to the content in small screens, that's more comfortable for the users. See reports:
<img width="1204" alt="Screenshot 2024-07-16 at 12 49 11 PM" src="https://github.com/user-attachments/assets/10f5b6bd-605e-4cbf-b933-e9c5b91e697e">

<img width="1091" alt="Screenshot 2024-07-16 at 12 50 17 PM" src="https://github.com/user-attachments/assets/149898d6-1fd9-4472-9c47-af8b2eb93dc4">